tasks/conf.yml

```diff
@@ -64,21 +64,18 @@
   tags:
     - config-additional
 
-- name: Upstart check
-  stat: path=/etc/init/
-  register: etc_init_check
-
-- name: systemd check
-  stat: path=/usr/lib/systemd/system/
-  register: systemd_check
-
 - name: Upstart environment variables
   lineinfile: dest=/etc/init/marathon.conf backup=yes state=present create=true insertbefore='exec.*' line="env {{ item }}"
   with_items: "{{ marathon_env_vars }}"
-  when: etc_init_check.stat.exists == true
+  when:
+    - ansible_service_mgr != 'systemd'
   notify: Restart marathon
 
+- name: Create /etc/sysconfig directory
+  file: path=/etc/sysconfig state=directory owner=root group=root mode=0755
```

Comment:
That `sysconfig` folder seems to be outdated (was only used in redhat-style OS, says ->link), then I'm wondering if those env-vars (by default only JAVA_OPTS) are used at all on Ubuntu systemd.
The new recommended way (both redhat/ubuntu) seems to be in `/etc/systemd/system/<service-name>.service.d/<something>.conf`
Resf:
* http://serverfault.com/questions/413397/how-to-set-environment-variable-in-systemd-service#answer-413408
* https://github.com/docker/docker/issues/9889#issuecomment-109819996